src/wasapi.h

```diff
@@ -13,14 +13,13 @@
 #include "list.h"
 #include "atomics.h"
 
-#define INITGUID
 #define CINTERFACE
 #define COBJMACROS
+#ifndef WIN32_LEAN_AND_MEAN
```

Comment:
I think I just got sick with the "already defined" warning at some point, and fixed it. I don't think it's necessary to do it for other macros unless it warning appears for them at some point.